packages/jest-snapshot/src/index.js

```diff
@@ -72,6 +107,7 @@ const toMatchSnapshot = function(
       : currentTestName || '';
 
   if (typeof propertyMatchers === 'object') {
+    const propertyMatchers = propertyMatchers;
```

Comment:
Why?

Reply:
Already removed locally, I had some argument shuffling happening earlier that I didn't completely clean up.